mysql-test/suite/galera/t/mdev_21718.test

```diff
@@ -0,0 +1,33 @@
+#
+# MDEV-21718 Reproduce a case where BF abort after
+# client session acquires the ownership but before calls
+# before_command() causes an assertion in wsrep-lib.
+#
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
```

Comment:
You are using debug sync so this needs --source include/have_debug.inc and --source include/have_debug_sync.inc, I will add them on merge.